multiversx_sdk_core/transaction_factories/delegation_factory.py

```diff
@@ -90,7 +82,7 @@ def add_nodes(self,
             sender=sender,
             receiver=delegation_contract,
             data_parts=parts,
-            execution_gas_limit=self.config.delegation_ops + num_nodes * self.config.additional_gas_limit_per_node,
+            execution_gas_limit=self.config.gas_limit_delegation_operations + num_nodes * self.config.additional_gas_limit_per_validator_node,
```

Comment:
Optional:

`self.compute_gas_for_add_nodes(num_nodes)`

Reply:
Fixed in https://github.com/multiversx/mx-sdk-py-core/pull/29.